thrift.js

```diff
@@ -394,13 +368,13 @@ Thrift.prototype.resolveMapConst = function resolveMapConst(def) {
     return map;
 };
 
-Thrift.prototype.resolveIdentifier = function resolveIdentifier(def, identifier, models) {
+Thrift.prototype.resolveIdentifier = function resolveIdentifier(def, name, models) {
     var self = this;
     var model;
 
     // short circuit if in global namespace of this thrift.
-    if (self.definitions[identifier]) {
-        model = self.definitions[identifier].link(self);
+    if (self.models[name]) {
+        model = self.models[name].link(self);
```

Comment:
This is a name fix. "Definitions" are what you get out of the Thrift IDL AST, like "fieldDef", and "def" elsewhere in this package. Thrift* objects are Models, previously called Specs.